.github/workflows/ci.yml

```diff
@@ -110,34 +110,34 @@ jobs:
           key: ${{ matrix.os }}-${{ hashFiles('submodules-status') }}-${{ matrix.unity-version }}-${{ matrix.unity-version-changeset }}-${{ hashFiles('package/package.json') }}
 
       - name: Setup Unity
-        uses: kuler90/setup-unity@f44e3f35ce3737316a15321ec3468161940e23e8
+        uses: bitsandfoxes/setup-unity@465111b8a4184a1ee4549e26cae0638b0df34f48
         id: setup-unity
         with:
           unity-version: ${{ matrix.unity-version }}
           unity-version-changeset:  ${{ matrix.unity-version-changeset }}
 
       - name: Setup Unity Modules - Windows IL2CPP
         if: matrix.os == 'windows-latest'
-        uses: kuler90/setup-unity@f44e3f35ce3737316a15321ec3468161940e23e8
+        uses: bitsandfoxes/setup-unity@465111b8a4184a1ee4549e26cae0638b0df34f48
```

Comment:
Is this change related to this PR? I don't follow why this change was made😅

Reply:
Not related to this PR, superseded by https://github.com/getsentry/sentry-unity/pull/510